examples/perf/ipc.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/python
+# IPC - Instructions Per Cycles using Perf Events and
+# uprobes
+# 24-Apr-2020	Saleem Ahmad	Created this.
+
+from __future__ import print_function
+from bcc import BPF, USDT, Perf
+from bcc.utils import printb
+from time import sleep
+import sys
+from optparse import OptionParser
+
+# load BPF program
+code="""
+#include <uapi/linux/ptrace.h>
+
+struct perf_delta {
+    u64 clk_delta;
+    u64 inst_delta;
+    u64 time_delta;
+};
+
+// Perf Arrays to read counter values for open
+// perf events.
+BPF_PERF_ARRAY(clk, 16);
+BPF_PERF_ARRAY(inst, 16);
```

Comment:
As a default, this might be too small, may be we can change to 128 cpus? And add a comment to remind user to increase max_entries if their system has more cpus.

Reply:
Added comment for the maximum number of cpu.